components/compositing/webview_renderer.rs

```diff
@@ -88,8 +88,8 @@ pub(crate) struct WebViewRenderer {
     touch_handler: TouchHandler,
     /// "Desktop-style" zoom that resizes the viewport to fit the window.
     pub page_zoom: Scale<f32, CSSPixel, DeviceIndependentPixel>,
-    /// "Mobile-style" zoom that does not reflow the page.
-    viewport_zoom: PinchZoomFactor,
+    /// "Mobile-style" zoom that does not reflow the page, i.e. pinch zoom.
```

Comment:
```suggestion
    /// "Mobile-style" zoom that does not reflow the page.
```
Let me remove this since the variable name already says pinch zoom...